### PR TITLE
Goofmed Bugfix: Screaming no longer kills you. God. Why. Come on.

### PR DESCRIPTION
**Pull request by Iamgoofball**
doctors were suffocating people with pain screams by accident

i don't need to explain why this is bad

### CODE REVIEW

**Comment by Iamgoofball** [comment]
according to gitblame this was an oldbase thing

**Comment by SpaceVampire** [comment]
This seems like it could be another reason for people to want to use painkillers.
Isn't that a good thing?

**Comment by ZephyrTFA** [comment]
Doctors should already be using painkillers or putting their patient to sleep using sleeping gas.
This is a bad change.

**Comment by Iamgoofball** [comment]
You two don't understand the actual problem here. The scream emote should not be killing people given the amount of ways you can force someone to *scream.

**Comment by ZephyrTFA** [comment]
> You two don't understand the actual problem here. The scream emote should not be killing people given the amount of ways you can force someone to *scream.

Its a grand total of 5 oxy damage.
It doesnt apply lose breath stacks, it doesnt apply nobreathe, it just gives a flat 5 oxy.
You are making a problem where there isn't one.

File: modular_skyrat/modules/emotes/code/scream_emote.dm [hunk 1]
```diff
@@ -7,8 +7,6 @@
 	if(!(. = ..()))
 		return
 	if(!user.is_muzzled() && !(user.mind && user.mind.miming))
-		if(ishuman(user))
-			user.adjustOxyLoss(5)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/emotes/code/scream_emote.dm:11]: Have it be stamina loss instead, bam, issue fixed.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by Gandalf2k15** [comment]
This was added on citadel to prevent scream spamming. We have a literal hard cooldown on emotes which includes screaming. So damage is no longer required.

**Pull request merged**